kube-metrics-adapter/templates/deployment.yaml

```diff
@@ -49,14 +53,24 @@ spec:
         volumeMounts:
         - mountPath: /tmp
           name: tmp
+        - name: ssl-certs
```

Comment:
Could you please make this option as well `if .Values.aws.enable`

Reply:
Done, Thank you for the review.